### PR TITLE
Feature/protocol flow

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -163,7 +163,7 @@
     "webpack-dev-server": "2.4.2"
   },
   "build": {
-    "appId": "com.ethereumclassic.emeraldwallet",
+    "appId": "com.ethereumclassic.emeraldwallet.test",
```
Comment (shanejonas) [inline package.json]: dont change `appId`

Comment (BelfordZ) [reply package.json]: 👍

File: electron/mainWindow.js [hunk 2]
```diff
@@ -4,12 +4,14 @@ const darwinMenu = require('./menus/darwin');
...
 const url = require('url');
 const devtron = require('devtron');
+const log = require('./logger');
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 let mainWindow;
 let menu;
 
+
```
Comment (BelfordZ) [inline electron/mainWindow.js]: remove this

File: electron/mainWindow.js [hunk 3]
```diff
@@ -26,11 +28,11 @@ const createWindow = function (openDevTools) {
     mainWindow.webContents.openDevTools();
     devtron.install();
   }
-  // Prevent opening external links in electron.
-  mainWindow.webContents.on('will-navigate', (e, _url) => {
-    e.preventDefault();
-    electron.shell.openExternal(_url);
-  });
+
+  /* mainWindow.webContents.on('will-navigate', (e, _url) => {
```
Comment (BelfordZ) [inline electron/mainWindow.js]: uncomment

File: src/store/store.js [hunk 4]
```diff
@@ -75,7 +82,7 @@ function refreshAll() {
 
   if (state.launcher.getIn(['geth', 'type']) === 'local') {
     promises = promises.concat([
-      store.dispatch(network.actions.loadPeerCount()),
+      /* store.dispatch(network.actions.loadPeerCount()),*/
```
Comment (shanejonas) [inline src/store/store.js]: can delete this

File: electron/mainWindow.js (hunk 3 above)
Comment (BelfordZ) [reply electron/mainWindow.js]: and delete

File: electron/protocol/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  .getAllWebContents()
+  .find((webcontent) => !!webcontent.browserWindowOptions);
+
+function protocolHandler(event, url) {
+  if (event) { event.preventDefault(); }
+  // need to ignore other windows such as chrome dev tools
+  const wc = getMainWebContents();
+
+  if (!wc) {
+    log.info('not sure we ever actually hit this anymore');
```
Comment (BelfordZ) [inline electron/protocol/index.js]: should remove this -- pretty sure its not used. I havent seen anyt of these logs

Comment (BelfordZ) [reply electron/protocol/index.js]: which means also taking out the timeout

File: src/components/tx/SendTx/SignTx/SignTxForm.js [hunk 6]
```diff
@@ -50,9 +51,40 @@ const displayFlexCenter = {
   alignItems: 'center',
 };
 
+
+const TypedData = (props) => {
+  const { typedData } = props;
+  if (!typedData) { return null; }
+  console.log('typedData', typedData);
```
Comment (BelfordZ) [inline src/components/tx/SendTx/SignTx/SignTxForm.js]: remove console

File: src/store/store.js (hunk 4 above)
Comment (BelfordZ) [inline src/store/store.js]: dont remember why we did this

File: src/store/store.js [hunk 7]
```diff
@@ -166,51 +169,44 @@ export const start = () => {
...
-          } else {
-            store.dispatch(screen.actions.gotoScreen('landing'));
-          }
-        });
-      }
+function loadInitalScreen() {
+  startSync();
+  const state = store.getState();
+  // If not first run, go right to home when ready.
+  store.dispatch(accounts.actions.loadAccountsList()).then(() => {
```
Comment (BelfordZ) [inline src/store/store.js]: Should possibly put this behind the trigger 'onceAccountsLoaded`

File: electron/vault/launcher.js [hunk 8]
```diff
@@ -77,6 +77,7 @@ class LocalConnector {
       const bin = this.emeraldExecutable();
       const appData = (process.env.APPDATA || os.homedir());
       const emeraldHomeDir = `${appData}${path.join('/.emerald', this.chain.name, 'keystore/')}`;
+      console.log(`balls${emeraldHomeDir}`);
```
Comment (BelfordZ) [inline electron/vault/launcher.js]: remove this

File: electron/mainWindow.js [hunk 9]
```diff
@@ -52,6 +47,7 @@ const createWindow = function (openDevTools) {
   return mainWindow.webContents;
 };
 
+
```
Comment (BelfordZ) [inline electron/mainWindow.js]: rm

File: electron/mainWindow.js [hunk 10]
```diff
@@ -4,6 +4,7 @@ const darwinMenu = require('./menus/darwin');
 const path = require('path');
 const url = require('url');
 const devtron = require('devtron');
+const log = require('./logger');
```
Comment (BelfordZ) [inline electron/mainWindow.js]: rm

**Pull request merged**